kernel/arch/dreamcast/include/dc/syscalls.h

```diff
@@ -0,0 +1,395 @@
+/* KallistiOS ##version##
+
+   dc/syscalls.h
+   Copyright (C) 2024 Andy Barajas
+*/
+
+/** \file      dc/syscalls.h
+    \brief     Functions to access the system calls of the Dreamcast ROM.
+    \ingroup   system_calls
+
+    \author Marcus Comstedt
+    \author Andy Barajas
+*/
+
+/** \defgroup  system_calls System Calls
+    \brief     API for the Dreamcast's system calls
+    \ingroup   system
+
+    This module encapsulates all the system calls available in the Dreamcast 
+    Operating System, allowing direct interaction with system hardware 
```

Comment:
```suggestion
    BIOS, allowing direct interaction with system hardware 
```
I dunno, I feel like calling it an "operating system" is a bit much, because it's more like a disjoint set of firmware routines than an actual OS. Also my first question as a newb is where the hell are these syscalls located, and you should probably at least mention that it's a part of the BIOS image.

Reply:
I agree, this is definitely not an OS.
16KB of syscalls loaded to begin of RAM from BIOS at startup:
http://www.dc-swat.ru/forum/thread-3799-post-41069.html#pid41069